stats/rpc_constants.textpb

```diff
@@ -0,0 +1,770 @@
+# Constants for collecting rpc stats.
+rpc_constants {
+  # Census defined rpc client MeasurementDescriptors.
+  measurement_descriptor {
+    name: "/rpc/client/error_count"
+    description: "RPC Errors"
```

Comment:
Descriptions should be longer, e.g. "Count of client RPC's that received an error (unsuccessful) response"